README.md

```diff
@@ -1,10 +1,58 @@
 # cosmoz-moment
 
 [![Build Status](https://travis-ci.org/Neovici/cosmoz-moment.svg?branch=master)](https://travis-ci.org/Neovici/cosmoz-moment)
+[![Published on webcomponents.org](https://img.shields.io/badge/webcomponents.org-published-blue.svg)](https://www.webcomponents.org/element/owner/my-element)
 
 cosmoz-moment is a Polymer component for centralized management of Moment.js with locale change distributed notification
 
-## Credits
+## Example
+
+<!---
+```
+<custom-element-demo>
+  <template>
+    <script src="../webcomponentsjs/webcomponents-lite.js"></script>
+	<link rel="import" href="../neon-animation/web-animations.html">
+	<link rel="import" href="../paper-dropdown-menu/paper-dropdown-menu.html">
+	<link rel="import" href="../paper-listbox/paper-listbox.html">
+	<link rel="import" href="../paper-item/paper-item.html">
+    <link rel="import" href="cosmoz-moment.html">
+    <link rel="import" href="cosmoz-moment-timeago.html">
+	<style is="custom-style">
```

Comment:
l added it for extra space on bottom cause dropdown list will have scrollbar in webcomponents preview. it's not critical and l will remove it